src/main/java/de/muenchen/isi/domain/service/AbfrageStatusService.java

```diff
@@ -1,59 +1,274 @@
+        this.sendEvent(id, StatusAbfrageEvents.FREIGABE, stateMachine);
+    }
+
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#ABBRUCH}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void abbrechenAbfrage(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.ABBRECHEN, stateMachine);
+    }
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#ANGELEGT}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void angabenAnpassenAbfrage(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.ANGABEN_ANPASSEN, stateMachine);
+    }
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#IN_ERFASSUNG}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void weitereAbfragevariantenAnlegen(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.WEITERE_ABFRAVARIANTEN_ANLEGEN, stateMachine);
+    }
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#IN_BEARBEITUNG_PLAN}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void keineZusaetzlicheAbfragevariante(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.KEINE_ZUSAEZTLICHE_ABFRAGEVARIANTE, stateMachine);
+    }
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#IN_ERFASSUNG}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
```

Comment:
Anmerkung um es einheitlich mit bereits bestehenden Javadocs zu machen.

```suggestion
     * @throws EntityNotFoundException falls die Abfrage nicht gefunden wird
```